actions/actions_whitebox_test.go

```diff
@@ -0,0 +1,176 @@
+package actions
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-wit/gormtestsupport"
+	"github.com/fabric8-services/fabric8-wit/resource"
+	tf "github.com/fabric8-services/fabric8-wit/test/testfixture"
+	"github.com/fabric8-services/fabric8-wit/workitem"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestSuiteAction(t *testing.T) {
+	resource.Require(t, resource.Database)
+	suite.Run(t, &ActionSuite{DBTestSuite: gormtestsupport.NewDBTestSuite()})
+}
+
+type ActionSuite struct {
+	gormtestsupport.DBTestSuite
+}
+
+func createWICopy(wi workitem.WorkItem, state string, boardcolumns []interface{}) workitem.WorkItem {
+	var wiCopy workitem.WorkItem
+	wiCopy.ID = wi.ID
+	wiCopy.SpaceID = wi.SpaceID
+	wiCopy.Type = wi.Type
+	wiCopy.Number = wi.Number
+	wiCopy.Fields = map[string]interface{}{}
+	for k := range wi.Fields {
+		wiCopy.Fields[k] = wi.Fields[k]
+	}
+	wiCopy.Fields["system.state"] = state
+	wiCopy.Fields["system.boardcolumns"] = boardcolumns
+	return wiCopy
+}
+
+func (s *ActionSuite) TestChangeSet() {
+	fxt := tf.NewTestFixture(s.T(), s.DB, tf.CreateWorkItemEnvironment(), tf.WorkItems(2))
+	require.NotNil(s.T(), fxt)
```

Comment:
This is done for you by the `tf.NewTestFixture`.

Reply:
I've seen shit happening there. An additional check is not a bad thing.

Reply:
If `fxt` is really `nil` then your check will not be reached. Just saying.

Reply:
Fixed.